lib/manager/cake/index.ts

```diff
@@ -17,15 +17,23 @@ const lexerStates = {
     dependency: {
       match: /^#(?:addin|tool|module)\s+(?:nuget|dotnet):.*$/,
     },
+    dependencyQuoted: {
+      match: /^#(?:addin|tool|module)\s+"(?:nuget|dotnet):[^"]+"\s*$/,
```

Comment:
We should add a moo [transform}(https://github.com/no-context/moo#value-vs-text) function, so we don't need to care about the quotes later

```suggestion
      match: /^#(?:addin|tool|module)\s+"(?:nuget|dotnet):[^"]+"\s*$/,
      value: s => s.slice(1, -1),
```

Reply:
```ts
s.replace(/"\s*$/, '')
```

Reply:
We also can remove `#prefix nuget:` part, though we may want to extract dep type some time later

Reply:
love it. Seems I need to read up on moo 😄 
But simply slicing leads to errors for trailing spaced after the last quote, right?

Reply:
Yes, so needs trim before slice.